feature/foryou/src/main/kotlin/com/google/samples/apps/nowinandroid/feature/foryou/ForYouScreen.kt

```diff
@@ -444,6 +473,38 @@ fun TopicIcon(
     )
 }
 
+@Composable
+private fun ShowDialog(
+    onDismissRequest: ()-> Unit,
+    askNotificationPermission: () -> Unit
+) {
+    AlertDialog(
+        onDismissRequest = onDismissRequest,
+        title = {
+            Text(stringResource(id = R.string.grant_notification_permission))
+        },
+        text = {
+            Text(stringResource(id = R.string.grant_notification_permission_description))
+        },
+        confirmButton = {
+            NiaButton(
+                onClick = {
+                    askNotificationPermission()
+                    onDismissRequest()
+                },
+            ) {
+                Text(text = "OK")
+            }
+        },
+        dismissButton = {
+            NiaButton(onClick =  onDismissRequest ) {
+                Text(text ="Cancel")
```

Comment:
To maintain consistency and ease of localization, the "Ok" and the "Cancel" strings also should be included in the string file.